src/kirby/components/tabs/tab-button/tab-button.component.ts

```diff
@@ -28,8 +29,13 @@ export class TabButtonComponent implements OnInit {
     this.removeWrapper();
   }
 
-  onClick(event: Event) {
+  onClick(event: Event, clickedSelected: boolean) {
     this.click.emit(event);
+
+    if (clickedSelected) {
+      const clickEvent = new CustomEvent(tabClicked);
```

Comment:
Seems a bit odd, that this event "tabClicked" is only dispatched if the tab was already selected??
Consider always dispatching event and adding a click-count that the event handler can take into account...

Reply:
Maybe rename event to `kirbyClickSelectedTab`?

Reply:
I think it is fair here to only dispatch if selected is clicked or it will add more complexity to the listener. Click count will not be enough as we might click different tabs. We would then need to parse the selectedTab id and then add a click count on that. I think just adding the event we actually need here will be an adequate solution. No big design upfront. :)

Reply:
As talked about I renamed the event with a more meaningful name and keep dispatching only if selected tab is clicked.